### PR TITLE
Use docker-compose to deploy

### PR DESCRIPTION
**Pull request by LiFengPro**
This PR includes:
1. Dockerfile for building the image of the website.
2. docker-compose file for developers to deploy the website in their own docker.
3. Update Readme.md to illustrate how to use docker-compose.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,35 @@
...
+
+2. Install docker-compose. Read more on [docker-compose installation](https://docs.docker.com/compose/install/)
+
+Command to run, just
+
+    # workdirs: /path/to/trazip
+    # build and pull images
+    docker-compose build
+    # start services
+    docker-compose up
```
Comment (SindraShen) [inline README.md]: Please describe how to access the running website.

Comment (SindraShen) [inline README.md:26]: Please describe how to access the running website.

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      dockerfile: ./Dockerfile
+    depends_on:
+    - db
+    networks:
+    - trazipnet
+    volumes:
+    - .:/src
+    environment:
+    - DEBUG=true
+    - DATABASE_TYPE=postgres
```
Comment (SindraShen) [inline docker-compose.yaml]: Please add some explanations about those variables.

**Review by SindraShen — COMMENTED** [review]
Please address my comments.

File: README.md (hunk 1 above)
Comment (LiFengPro) [reply README.md:26]: Done.

File: docker-compose.yaml (hunk 2 above)
Comment (LiFengPro) [reply docker-compose.yaml]: Done

**Review by SindraShen — APPROVED** [review]
Looks good to me.

**Pull request merged**